content/tutorials/ios-push-server-registration.textile

```diff
@@ -53,16 +53,14 @@ h2(#step8). Step 8 - Testing the app
 
 <%= partial 'shared/tutorials/_testing_push_ios' %>
 
-<a href="https://github.com/ably/tutorials/commit/974d278c64707a258cbda7f0ebf833c39e8dcc78">See this step in GitHub.<a/>
+<a href="https://github.com/ably/tutorials/commit/974d278c64707a258cbda7f0ebf833c39e8dcc78">See this step in GitHub.</a>
 
 h2(#further-reading). Further Reading
 
-* Find the complete source code for this project on <a href="https://github.com/ably/tutorials/tree/push-registration-via-server-ios">GitHub.<a/>
+* Find the complete source code for this project on <a href="https://github.com/ably/tutorials/tree/push-registration-via-server-ios">GitHub.</a>
 
 * Ably's "Push Notifications documentation":/general/push and info on "channels":/core-features/channels may also be useful.
 
 * "Apple's documentation":https://developer.apple.com/notifications/ is a good place to find more information on the use of Push Notifications.
 
-* You may also wish to deregister devices, information on that may be found in Ably's <a href="https://www.ably.io/documentation/rest-api#delete-device-registration">Push Notifications docs.</a>
-
-* Using the "Push Admin API":/general/push/admin, you can get all of the stats related to your application implementing Push Notifications, such as the number and type of device registrations, etc.
+* You may also wish to deregister devices or get all of the stats related to your application implementing Push Notifications, such as the number and type of device registrations, etc. More information on how to do this can be found in the <a href="/general/push/admin">Push Admin API docs.</a>
```

Comment:
`<a href="/general/push/admin">Push Admin API docs.</a>` to `"Push Admin API docs":/general/push/admin`